examples/exp_configs/rl/singleagent/singleagent_straightroad.py

```diff
@@ -0,0 +1,163 @@
+"""Multi-agent highway with ramps example.
+Trains a non-constant number of agents, all sharing the same policy, on the
```

Comment:
Can you make it so the expconfigs for SA and MA are both either {ma/sa}_straight_road.py or {ma/sa}_straightroad.py ?

Reply:
yeah good point